pages/organizations-and-projects/concepts.mdx

```diff
@@ -24,7 +24,7 @@ An Organization is made of one or several [Projects](#project). When you create
 
 ## Organization ID
 
-The Organization ID identifies the [Organization](#organization) created with your account. It can be found on your [Organization dashboard](https://console.scaleway.com/organization/settings), in the **Settings** tab.
+The Organization ID identifies the [Organization](#organization) created with your account. It can be found on your [Organization dashboard](https://console.scaleway.com/organization/settings), in the **Settings** tab. You can click the **Copy ID** button next to the Organization name to do so.
```

Comment:
```suggestion
The Organization ID identifies the [Organization](#organization) created with your account. It can be found on your [Organization dashboard](https://console.scaleway.com/organization/settings), in the **Settings** tab. You can also click the **Copy ID** button next to the Organization name to do so.
```